ossdbtoolsservice/metadata/metadata_service.py

```diff
@@ -22,6 +22,7 @@
 UNION
 SELECT schemaname AS schema_name, tablename AS object_name, 't' as type FROM pg_tables
     WHERE schemaname NOT ILIKE 'pg_%' AND schemaname != 'information_schema'
+      AND tablename NOT IN ( SELECT relname FROM pg_class WHERE relispartition )
```

Comment:
I worried this change might introduce inefficiency, but I verified it generates the same EXPLAIN plan as using a CTE for the subquery.